class-pinterest-for-woocommerce.php

```diff
@@ -671,20 +682,64 @@ public static function update_account_data() {
 						'verified_domains' => '',
 						'domain_verified'  => '',
 						'username'         => '',
+						'full_name'        => '',
 						'id'               => '',
 						'image_medium_url' => '',
+						'is_partner'       => '',
 					)
 				);
 
 				Pinterest_For_Woocommerce()::save_setting( 'account_data', $data );
 				return $data;
 			}
 
+			self::get_linked_businesses( true );
+
 			return array();
 
 		}
 
 
+		/**
+		 * Fetches a fresh copy (if needed or explicitly requested), of the authenticated user's Linked business accounts.
+		 *
+		 * @param boolean $force_refresh Wether to refresh the data from the API.
+		 *
+		 * @return array
+		 */
+		public static function get_linked_businesses( $force_refresh = false ) {
+
+			$linked_businesses = Pinterest_For_Woocommerce()::get_data( 'linked_businesses' );
+
+			if ( $force_refresh || is_null( $linked_businesses ) ) {
```

Comment:
Comparing directly to `null` is preferable to using `is_null()`. 
```suggestion
			if ( $force_refresh || null === $linked_businesses ) {
```